docs/en/Configuring-Alluxio-with-S3.md

```diff
@@ -36,16 +38,44 @@ To configure Alluxio to use S3 as its under storage system, modifications to the
 bucket and directory as the under storage system. You specify it by modifying `conf/alluxio-env.sh`
 to include:
 
-{% include Configuring-Alluxio-with-S3/underfs-address.md %}
+{% include Configuring-Alluxio-with-S3/underfs-address-s3n.md %}
 
-Next, you need to specify the AWS credentials for S3 access. In `conf/alluxio-env.sh`, add:
+or
+
+{% include Configuring-Alluxio-with-S3/underfs-address-s3a.md}
+
+Next, you need to specify the AWS credentials for S3 access.
+
+If you are using s3n, in `conf/alluxio-env.sh`, add:
 
 {% include Configuring-Alluxio-with-S3/aws.md %}
 
 Here, `<AWS_ACCESS_KEY_ID>` and `<AWS_SECRET_ACCESS_KEY>` should be replaced with your actual
 [AWS keys](https://aws.amazon.com/developers/access-keys), or other environment variables that
 contain your credentials.
 
+If you are using s3a, you can specify credentials in 4 ways, from highest to lowest priority:
+
+* Environment Variables `AWS_ACCESS_KEY_ID` or `AWS_ACCESS_KEY` and `AWS_SECRET_ACCESS_KEY` or
```

Comment:
not very clear what is the relationship between `AWS_ACCESS_KEY_ID` and `AWS_ACCESS_KEY`, also for `AWS_SECRET_ACCESS_KEY` and `AWS_SECRET_KEY`